packages/pds-ember/app/styles/pds/core/typography/stories/index.stories.js

```diff
@@ -0,0 +1,217 @@
+import hbs from 'htmlbars-inline-precompile';
+import DocsPage, { TITLE } from './docs.mdx';
+
+const CONFIG = {
+  title: TITLE,
+  parameters: { docs: { page: DocsPage } },
+};
+
+const Index = () => ({
+  template: hbs`
+    <section>
+      <header>
+        <h1 class="pds-heading pds--0">Heading 0</h1>
```

Comment:
Do you think there will be other places you use a scale class like this? (headers is the only one I can think of)

Reply:
Not currently, but the modifier classes are designed to be used in conjunction with another CSS class for style application (i.e., you can't use the modifier on its own; `.pds--0` doesn't do anything on its own).  The base class (`.pds-heading`) is necessary to provide a single class to be able to target all headings with minimal selectors.